salt/utils/event.py

```diff
@@ -586,6 +591,9 @@ def __del__(self):
             self.destroy()
         except Exception as ex:
             log.debug(ex)
+            # as this method can be called when logging is already teared down
+            # just try to stdout it
+            traceback.print_exc(file=sys.stderr)
```

Comment:
not really, when we are dealing with __del__, in py2, its hard to know what we can call without throwing out new exeptions which will shallow the real one.